src/services/F2fService.ts

```diff
@@ -209,12 +209,13 @@ export class F2fService {
 	async sendToIPVCore(event: IPVCoreEvent): Promise<void> {
 		try {
 			const messageBody = JSON.stringify(event);
+			const queueUrl = this.environmentVariables.getIpvCoreQueueURL(this.logger)
 			const params = {
 				MessageBody: messageBody,
-				QueueUrl: this.environmentVariables.getIpvCoreQueueURL(this.logger),
+				QueueUrl: queueUrl,
 			};
 
-			this.logger.info({ message: "Sending message to IPV Core Queue", messageBody });
+			this.logger.info({ message: `Sending message to IPV Core Queue ${queueUrl}`, messageBody });
```

Comment:
easier to search logs in CW or Splunk this way...
```suggestion
			this.logger.info({ message: "Sending message to IPV Core Queue", queueUrl, messageBody });
```